timescaledb/how-to-guides/hyperfunctions/time-bucket-gapfill.md

```diff
@@ -0,0 +1,17 @@
+# Time bucket gapfill
+Sorting data into time buckets works well in most cases, but problems can arise
+if there are gaps in the data. This can happen if you have irregular sampling
+intervals, or you have experienced an outage of some sort. If you have a time
+bucket that has no data at all, the average returned from the time bucket is
+NULL, which could cause problems. You can use a gapfilling function to create
+additional rows of data in any gaps, ensuring that the returned rows are in
+chronological order, and contiguous. The time bucket gapfill function creates a
```

Comment:
This appears to be the second thing on time bucket gapfill, not sure if wer'e meant to have both?